service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -7,14 +7,7 @@
 
 import com.codeforcommunity.api.ITeamsProcessor;
 import com.codeforcommunity.auth.JWTData;
-import com.codeforcommunity.dto.team.AddGoalRequest;
-import com.codeforcommunity.dto.team.CreateTeamRequest;
-import com.codeforcommunity.dto.team.GoalResponse;
-import com.codeforcommunity.dto.team.InviteUsersRequest;
-import com.codeforcommunity.dto.team.TeamDataResponse;
-import com.codeforcommunity.dto.team.TeamGoalDataResponse;
-import com.codeforcommunity.dto.team.TransferOwnershipRequest;
-import com.codeforcommunity.dto.team.UsersResponse;
+import com.codeforcommunity.dto.team.*;
```

Comment:
Same as above

Reply:
Got it that was done automatically but just added them all back

Reply:
If you want to disable auto-wildcard imports, [check this out](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports)